pkg/kn/commands/container/container_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2021 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package container
+
+import (
+	"bytes"
+	"testing"
+
+	"gotest.tools/v3/assert"
+	"knative.dev/client/pkg/kn/commands"
+)
+
+func TestContainerCommand(t *testing.T) {
+	knParams := &commands.KnParams{}
+	containerCmd := NewContainerCommand(knParams)
+	assert.Equal(t, containerCmd.Name(), "container")
+	assert.Equal(t, containerCmd.Use, "container COMMAND")
```

Comment:
this is more testing cobra :-) But I think its important to get to the required code coverage. One reason, why I think that code coverage alone is not the holy grail.

Reply:
Yep, entirely for the sake of test coverage. :)